Lib/datetime.py

```diff
@@ -1468,6 +1468,8 @@ def _cmp(self, other, allow_mixed=False):
                 raise TypeError("cannot compare naive and aware times")
         myhhmm = self._hour * 60 + self._minute - myoff//timedelta(minutes=1)
         othhmm = other._hour * 60 + other._minute - otoff//timedelta(minutes=1)
+        myhhmm %= (24 * 60)
```

Comment:
I dont know why we have two variable names pointing to the same data and they dont even change later? Am I missing something?

Reply:
I am guessing you misread what these lines are doing? These aren't assigning these variables to be `24*60`, it's assigning then to be the original value stored in each variable, modulo the number of minutes in a day. So if `myhhmm` is `86495` and `othhmm` is `95`, they'll both be `95` after this operation.

Reply:
I see, thanks for the clarity.